flink-connector-aws/flink-connector-aws-kinesis-streams/src/main/java/org/apache/flink/connector/kinesis/source/KinesisStreamsSource.java

```diff
@@ -254,31 +252,10 @@ private RetryStrategy createExpBackoffRetryStrategy(
         final BackoffStrategy backoffStrategy =
                 BackoffStrategy.exponentialDelayHalfJitter(initialDelay, maxDelay);
 
-        return StandardRetryStrategy.builder()
+        return SdkDefaultRetryStrategy.standardRetryStrategyBuilder()
                 .backoffStrategy(backoffStrategy)
                 .throttlingBackoffStrategy(backoffStrategy)
                 .maxAttempts(maxAttempts)
```

Comment:
Circuit breaker is enabled in this strategy by default. Is this intended?

Reply:
I was wondering that - is there a reason we really don't want to enable circuit breaker for our setup?

Reply:
I don't think so. I believe it will be useful, we just need to document this as a new change as it might be surprising for users.